contracts/superswap/RexSuperSwap.sol

```diff
@@ -0,0 +1,147 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+pragma abicoder v2;
+
+import { ISuperToken } from "@superfluid-finance/ethereum-contracts/contracts/interfaces/superfluid/ISuperfluid.sol";
+import "./TransferHelper.sol";
+import "./interfaces/ISwapRouter02.sol";
```

Comment:
We need a note here for future reference, 

`- You will manually have to change `@uniswap/swap-router-contracts/contracts/interfaces/ISwapRouter02.sol` `^0.7.6` ==> `^0.8.0``